src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+    String getPrintableString();
```

Comment:
Header comment missing here. Use Returns... as a description and omit @return line.